cabot/cabotapp/tests/test_web.py

```diff
@@ -1,11 +1,16 @@
 # -*- coding: utf-8 -*-
-
 from django.core.urlresolvers import reverse
+from django.db import IntegrityError
 from django.test.client import Client
 from datetime import timedelta, date
-from cabot.cabotapp.models import Service, HttpStatusCheck
+
+from django.utils import timezone
+from rest_framework.test import APITransactionTestCase
+
+from cabot.cabotapp.models import Service, HttpStatusCheck, ServiceStatusSnapshot
 from cabot.cabotapp.views import StatusCheckReportForm, ServiceListView
 from .utils import LocalTestCase
+import threading
```

Comment:
nit: would be nice to cleanup these imports https://www.python.org/dev/peps/pep-0008/#imports